definitions/procedures/backup/prepare_directory.rb

```diff
@@ -21,12 +22,16 @@ def run
       end
 
       FileUtils.rm(Dir.glob(File.join(@backup_dir, '.*.snar'))) if @preserve_dir
-      if @incremental_dir && !(snar_files = Dir.glob(File.join(@incremental_dir, '.*.snar'))).empty?
-        FileUtils.cp(snar_files, @backup_dir)
-      else
-        raise "Either #{@incremental_dir} or #{@incremental_dir}/*.snar files unavailable. "\
-              'Provide a valid previous backup directory'
+      if @incremental_dir
+        snar_files = Dir.glob(File.join(@incremental_dir, '.*.snar'))
+        if snar_files.empty?
+          raise "Either #{@incremental_dir} or #{@incremental_dir}/*.snar files unavailable. "\
```

Comment:
With this change we need to change the message ? because @incremental_dir should be available under the if block ? Something like below,
`#{@incremental_dir}/*.snar files unavailable. Provide a valid previous backup directory'.`